aviary/mission/gasp_based/ode/test/test_taxi_eom.py

```diff
@@ -17,21 +17,25 @@ class TaxiFuelComponentTestCase(unittest.TestCase):
     def setUp(self):
         self.prob = om.Problem(model=om.Group())
 
+        num_nodes = 2
+
         aviary_options = AviaryValues()
         aviary_options.set_val(Mission.Taxi.DURATION, 0.1677, units="h")
 
         self.prob.model.add_subsystem('taxi', TaxiFuelComponent(
-            aviary_options=aviary_options), promotes=['*'])
+            num_nodes=num_nodes, aviary_options=aviary_options), promotes=['*'])
 
     def test_fuel_consumed(self):
         self.prob.setup(force_alloc_complex=True)
 
-        self.prob.set_val(
-            Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL, -1512, units="lbm/h")
+        self.prob.set_val(Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL,
```

Comment:
I think I would use different values for the 2 elements in this test. I know this is a simple component, and will work fine, but it is helpful in catching more errors in general.

Reply:
Done.